pkg/cmd/clients_test.go

```diff
@@ -417,11 +417,48 @@ func TestMobileClientsCmd_TestCreateClient(t *testing.T) {
 				}
 			},
 		},
+		{
+			Name: "test create xamarin mobile client succeeds without error",
+			Args: []string{"test", "xamarin", "my.app.org"},
+			MobileClient: func() mc.Interface {
+				fkMc := &mcFake.Clientset{}
+				fkMc.AddReactor("create", "mobileclients", func(action kt.Action) (handled bool, ret runtime.Object, err error) {
+					ca := action.(kt.CreateAction)
+					return true, ca.GetObject(), nil
+				})
+				return fkMc
+			},
+			Flags: []string{"--namespace=myproject", "-o=json"},
+			Validate: func(t *testing.T, c *v1alpha1.MobileClient) {
+				if nil == c {
+					t.Fatal("expected a mobile client but got nil")
+				}
+				if c.Spec.ClientType != "xamarin" {
+					t.Fatal("expected the clientType to be cordova but got ", c.Spec.ClientType)
+				}
+				if c.Spec.AppIdentifier != "my.app.org" {
+					t.Fatal("expected an appIdentifier to be set as my.app.org but was  ", c.Spec.AppIdentifier)
+				}
+				if c.Spec.Name != "test" {
+					t.Fatal("expected the client name to be test but got ", c.Spec.Name)
+				}
+				if c.Spec.ApiKey == "" {
+					t.Fatal("expected an apiKey to be generated but it was empty")
+				}
+				icon, ok := c.Annotations["icon"]
+				if !ok {
+					t.Fatal("expected an icon to be set but there was none")
+				}
+				if icon != "fa fa-xamarin" {
```

Comment:
is this passing? It looks like it is a different value [here](https://github.com/aerogearcatalog/xamarin-app-apb/pull/1/files#diff-bd84d488df2d1137568929b31a1f9c71R10)?